src/fmu/tools/rms/generate_petro_jobs_for_field_update.py

```diff
@@ -13,19 +13,16 @@
 import copy
 import json
 import pprint
-import warnings
 
 import yaml
 
-try:
-    import _roxar  # type: ignore
-except ModuleNotFoundError:
-    try:
-        import _rmsapi as _roxar  # type: ignore
-        import roxar.jobs  # type: ignore
-        import roxar.jobs.Job  # type: ignore
-    except ModuleNotFoundError:
-        warnings.warn("This script only supports interactive RMS usage", UserWarning)
+from fmu.tools import ROXAR
+
+if ROXAR:
+    import rmsapi
```

Comment:
You would need to import the `rmsapi.jobs` explicit here to avoid an AttributeError.

![image](https://github.com/user-attachments/assets/1ea6dd7b-a34c-4a90-830f-845c794c3b21)